pkg/koordlet/metriccache/metric_cache.go

```diff
@@ -147,7 +149,7 @@ func (m *metricCache) GetNodeResourceMetric(param *QueryParam) NodeResourceQuery
 		return result
 	}
 	if len(metrics) == 0 {
-		result.Error = fmt.Errorf("get node resource metric not exist, query params %v", param)
+		result.Metric = &NodeResourceMetric{}
```

Comment:
will GetNodeResourceMetric also import error logs?

Reply:
`GetNodeResourceMetric `  also should ignore error log, maybe it's the same kind of problem as report-nodetopo